Cargo.toml

```diff
@@ -5,7 +5,7 @@ name = "mio"
 # When releasing to crates.io:
 # - Update CHANGELOG.md.
 # - Create git tag
-version       = "1.0.0"
+version       = "1.0.1"
```

Comment:
Up to you, but should this not be 1.1.0 due to the new OS support?

Reply:
I was thinking about using `1.x` for MSRV updates: https://github.com/tokio-rs/mio?tab=readme-ov-file#msrv-policy.